src/components/Code.jsx

```diff
@@ -45,7 +46,7 @@ function Code() {
                 <MemoryView loaded={wasmLoaded} registerCallback={registerCallback} />
             </div>
             <div className="fill">
-                <Console loaded={wasmLoaded} registerCallback={registerCallback} exitCode={exitCode} error={error} />
+                <Console loaded={wasmLoaded} registerCallback={registerCallback} exitCode={exitCode} error={error} state={state} stepBack={stepBack} setState={setState}/>
```

Comment:
console should not need to access step back

Reply:
@WyattRoss this one's you